src/python/grpcio_tests/tests/unit/resources.py

```diff
@@ -34,3 +34,71 @@ def private_key():
 def certificate_chain():
     return pkg_resources.resource_string(__name__,
                                          _CERTIFICATE_CHAIN_RESOURCE_PATH)
+
+
+# the following are two disjoint certificate hierarchies 1 and 2,
```

Comment:
Drop this; we generally dislike "stanza" markings in our source code (with the solution being to reshape the code so that no module needs "stanzas".) (Yes, the `grpc/__init__.py` file is an exception to this, although we're not thrilled with the stanza markings there either...)

Reply:
will do

Reply:
done